packages/node_modules/@node-red/editor-client/src/js/ui/workspaces.js

```diff
@@ -130,6 +178,28 @@ RED.workspaces = (function() {
                         $("#red-ui-tab-"+(workspace.id.replace(".","-"))).toggleClass('red-ui-workspace-disabled',!!workspace.disabled);
                         $("#red-ui-workspace").toggleClass("red-ui-workspace-disabled",!!workspace.disabled);
 
+                        var old_env = workspace.env;
+                        var new_env = RED.subflow.exportSubflowInstanceEnv(workspace);
+                        console.log("; NE", new_env);
+                        if (new_env && (new_env.length > 0)) {
+                            new_env.forEach(function(prop) {
+                                if (prop.type === "cred") {
+                                    workspace.credentials = workspace.credentials || {_:{}};
```

Comment:
Have you tested to see if flow/group credential types get stored properly? I can't immediately see any code to handle loading credentials for flows/groups.

Reply:
I was missing the test for credential. Thank you for pointing this out.
Please let me know if there is anything strange in the fix.